src/workspaces/dashboard/WorkspaceTags.ts

```diff
@@ -90,12 +90,12 @@ export const WorkspaceTags = (props: WorkspaceTagsProps): ReactNode => {
         ]),
         canEdit &&
           div({ style: { marginBottom: '0.5rem' } }, [
-            h(WorkspaceTagSelect, {
+            h(WorkspaceTagSelect<false>, {
```

Comment:
Can you convert this file to TSX?

Reply:
Actually, since it was already ts to start I guess it is out-of-scope for this PR.